src/Rules/ForbiddenMultipleClassLikeInOneFileRule.php

```diff
@@ -11,11 +11,13 @@
 use PHPStan\Analyser\Scope;
 use PHPStan\Node\FileNode;
 use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
 use Symplify\RuleDocGenerator\Contract\DocumentedRuleInterface;
 use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
 use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
 
 /**
+ * @implements <FileNode>
```

Comment:
The autocompletion does not work because this should say `@implements Rule<FileNode>`.

Reply:
Thanks!